.cirrus.yml

```diff
@@ -724,16 +588,14 @@ success_task:
         - "build_without_cgo"
         - "container_image_build"
         - "meta"
-        - "image_prune"
+        #- "image_prune"
```

Comment:
I'd feel much better if this was removed, since the corresponding `_task` was also removed

Reply:
woops.